### PR TITLE
Add note for setting up poetry in a pre-existing project.

### PR DESCRIPTION
**Pull request by 8F3E**
# Pull Request Check List

Resolves: #2630

- [ ] Added **tests** for changed code.
- [ ] Updated **documentation** for changed code.

No code was modified; only documentation changed.

### CODE REVIEW

File: docs/docs/basic-usage.md [hunk 1]
```diff
@@ -41,6 +41,15 @@ python = "*"
...
 
+If you have a pre-existing project, say `pre-existing-demo`, and want to use poetry:
+
+```bash
+cd pre-existing-demo
+poetry init
+```
+
+This will interactively guide you through creating a `pyproject.toml` file.
+
```
Comment (finswimmer) [inline docs/docs/basic-usage.md:52]: Thanks a lot your contribution @8F3E :+1: 

It's a good idea to show the `poetry init` way in the "basic usage" as well. But I think it should be separate headline and should include some words about the folder structure that `poetry` expects.

fin swimmer

Comment (8F3E) [reply docs/docs/basic-usage.md]: I've added the new headline and a bit about folder structure, not quite sure if it's in the right place in the file though.

File: docs/docs/basic-usage.md [hunk 2]
```diff
@@ -41,6 +41,24 @@ python = "*"
...
+pre-existing-project
+├── README.rst
+└── pre_existing_project
+    └── __init__.py
+```
+
+To initialise this project, and interactively create a `pyproject.toml` file:
+
+```bash
+poetry init pre-existing-project
```
Comment (finswimmer) [inline docs/docs/basic-usage.md:59]: Thanks for the changes!

One more note: `poetry init` doesn't expects the folder name here. Just use `poetry init` within the root folder.

Comment (8F3E) [reply docs/docs/basic-usage.md]: Oops! Hopefully this is better now.

**Review by finswimmer — APPROVED** [review]
LGTM 👍 

Thanks a lot!

**Comment by sburris0** [comment]
When do the docs update? This change isn't displaying on the website.

**Pull request merged**